api/controllers/EventController.js

```diff
@@ -27,12 +30,37 @@ var EventController = {
 
                 delete event.lng;
                 delete event.lat;
-
-                Event.create(event, function (err, _) {
-                    if (err) return res.status(500).json(err.message);
-
-                    return res.status(200).json(_._id);
-                })
+                
+                superagent
```

Comment:
I am not sure how to use our existing API to get restaurant detailed info.

Reply:
Maybe take the `getRestaurantById` into a method on its own and it can be accessed by both event creation and GET restaurant endpoint

Reply:
Sounds good